modules/google/google_artifact_registry_repository_iam/main.tf

```diff
@@ -2,36 +2,36 @@
 
 resource "google_artifact_registry_repository_iam_binding" "binding" {
   repository = var.repository
+  location   = var.location
   count      = var.module_enabled && var.policy_bindings == null && var.authoritative ? 1 : 0
   depends_on = [
     var.module_depends_on,
   ]
-  location = var.location
```

Comment:
@i4ki terramate generate seems to not respect order (alphabetical) when mixing attributes and content blocks... is this intended? ;)